app/views/shared/_header_menu.html.erb

```diff
@@ -0,0 +1,6 @@
+<ul class="nav nav-background nav-txt-color p-2">
+  <a class="navbar-brand nav-txt-color" style="color: white" href="/">Github Analyzer</a>
```

Comment:
Let's avoid inline styles in the `HTML` markup, unless they are critical ones loaded in the layout to improve performance over unused ones above the fold.

What do you think of creating a `CSS` file called `_nav_bar.scss` (or similar) inside `stylesheets/components/`, where you can declare all styles related to this Nav Bar?